### ISSUE TITLE
Add "Double dice roll on critical" option to critical hit rules.

### ISSUE THREAD

**Issue by cbprosser** [kakaroto/Beyond20#144]
I (and many others due to Critical Role) use the homebrew critical hit option to double the number shown on the dice on a critical hit (not doubling the modifiers). It would be nice to have that option.

For instance: I deal 1d4+3 damage on hit with a dagger as a rogue. I score a critical, rolling 1d4 dagger damage as well as 2d6 sneak attack. I roll 3, 4, and 6 on the die, making my damage 2*3 + 2*4 + 2*6 + 3 = 29 damage.

**Comment by kakaroto** [comment]
Hey. Unfortunately that option isn't possible because of Roll20. There is no way to get the rolled dice value or to tell it to double it the value of a previous roll or anything of the sort.
I've briefly mentioned the limitation in the [v0.7 release notes](https://beyond20.here-for-more.info/release_notes#v07) when I added the homebrew options.
If you find a way to tell Roll20 to do it with a macro, let me know, otherwise it's not going to happen.
(it would be possible with Foundry VTT, but I don't remember why I didn't add the option just for it)

**Comment by cbprosser** [comment]
On Roll20, you can do a /gmroll (or /roll) like this:

/gmroll 1d20*2 and it will multiply the roll.

**Comment by kakaroto** [comment]
I know that you can use a multiplication in the rolls. But you can't say roll 2d6 and in case of a crit, use the same result from that 2d6 and multiply it by 2.
The problem here is that you don't know if it's a crit or not until the roll is made, so it's too late to multiply it, it's a single command for all the rolls. Do a roll with beyond20, then in Roll20 press the up arrow on the chat to see the actual command that beyond20 sends. You'll see there's an entry for normal damage and an entry for critical damage. they are two separate formulas and you can't reuse the same dice rolls from one formula to the next.
